app/repository/AdminDataUnitLinksEditRepository.scala

```diff
@@ -11,5 +11,5 @@ case object EditFailure extends EditParentLinkStatus
 case object EditConflict extends EditParentLinkStatus
 
 trait AdminDataUnitLinksEditRepository {
-  def patchVatParentUnitLink(patch: Patch, period: Period, vatref: VatRef): Future[EditParentLinkStatus]
+  def patchVatParentUnitLink(patches: Seq[Patch], period: Period, vatref: VatRef): Future[EditParentLinkStatus]
```

Comment:
It does not feel as if this signature is adequate to describe the actions required.
More on this later.